### DISCUSSION TITLE
Error "No such DSL method 'terrform' found among steps".

### DISCUSSION THREAD

**Discussion by Pappik** [learndevopsonline/learndevopsonline#1665] [Q&A] [answered]
<img width="750" alt="Screenshot 2023-12-07 171042" src="https://github.com/learndevopsonline/learndevopsonline/assets/89238944/d84f793e-5cc0-43ef-89a1-2d6da64ed86b">
while trying to run a job 'roboshop-infra' getting this error. tried to install terraform plugins in jenkins but not helpful.

**Comment by sreedharm07** [comment, marked as answer]
your file name is **terraform** but 
you are calling **terrform**
change them and run 



![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/0e76549f-51a3-4a0e-92f1-37399384f3e0)![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/f9b3589b-aabf-49e7-a40b-03113f6f7444)

**Reply by Pappik** [reply]
i overlooked it. tq @sreedharm07
